app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -714,6 +715,14 @@ class AdministratorControlsActivityTest {
     assertThat(screenName).isEqualTo(ScreenName.ADMINISTRATOR_CONTROLS_ACTIVITY)
   }
 
+  @Test
+  fun testActivity_createIntent_verifyProfileIdInIntent() {
```

Comment:
Can we also add a regex check similar to the one we added for screen name to ensure all activity tests have this test present?

Reply:
I have already added this check to activities using ProfileId.

Reply:
Updated this thanks.